src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -6,192 +6,186 @@ import { Callout } from "/src/components/callout"
+
+3. Create a `config.toml` file and put it in your `~/.ampd/` directory. 
+
+By default, `ampd` loads the `config` file from `~/.ampd/config.toml` when running any command. This can be overridden by passing `--config [path]`.
+
+The file should have the following format:
+
+```toml
+tm_jsonrpc=[JSON-RPC URL of Axelar node]
+tm_grpc=[gRPC URL of Axelar node]
+event_buffer_cap=[max blockchain events to queue. Will error if set too low]
+
+[service_registry]
+cosmwasm_contract=[address of service registry]
+
+[broadcast]
+batch_gas_limit=[max gas for a transaction. Transactions can contain multiple votes and signatures]
+broadcast_interval=[how often to broadcast transactions]
+chain_id=[chain id of Axelar network to connect to]
+gas_adjustment=[gas adjustment to use when broadcasting]
+gas_price=[gas price with denom, i.e. "0.007uaxl"]
+queue_cap=[max messages to queue when broadcasting]
+tx_fetch_interval=[how often to query for transaction inclusion in a block]
+tx_fetch_max_retries=[how many times to query for transaction inclusion in a block before failing]
+
+[tofnd_config]
+key_uid=[uid of key used for signing transactions]
+party_uid=[metadata, should just be set to ampd]
+url=[url of tofnd]
+
+# multisig handler. This handler is used for all supported chains.
+[[handlers]]
+cosmwasm_contract=[address of multisig contract]
+type="MultisigSigner"
+
+# message verifier handler. One per supported chain
+[[handlers]]
+chain_name=[chain name. Not necessary in the Sui case]
+chain_rpc_url=[URL of JSON-RPC endpoint for external chain]
+cosmwasm_contract=[verifier contract address]
+type=[handler type. Could be EvmMsgVerifier | SuiMsgVerifier]
+
+# workerset verifier handler. One per supported chain
+[[handlers]]
+chain_name=[chain name. Not necessary in the Sui case]
+chain_rpc_url=[URL of JSON-RPC endpoint for external chain]
+cosmwasm_contract=[verifier contract address]
+type=[handler type. Could be EvmMsgWorkerSetVerifier | SuiWorkerSetVerifier]
+```
 
-- If you are running `tofnd` for the first time, a verifier key will be generated for you automatically.
+Here's an example config for connecting to a local axelard node and `tofnd` process and verifying transactions from the Avalanche and Sui testnets:
+
+```toml
+tm_jsonrpc="http://localhost:26657"
+tm_grpc="tcp://localhost:9090"
+event_buffer_cap=10000
+
+[service_registry]
+cosmwasm_contract="axelar1hrpna9v7vs3stzyd4z3xf00676kf78zpe2u5ksvljswn2vnjp3ystlgl4x"
+
+[broadcast]
+batch_gas_limit="10000000"
+broadcast_interval="1s"
+chain_id="devnet-amplifier"
+gas_adjustment="2"
+gas_price="0.00005uamplifier"
+queue_cap="1000"
+tx_fetch_interval="600ms"
+tx_fetch_max_retries="10"
+
+[tofnd_config]
+key_uid="axelar"
+party_uid="ampd"
+url="http://127.0.0.1:50051"
+
+[[handlers]]
+type = 'MultisigSigner'
+cosmwasm_contract = 'axelar1au3qdftlypz3uydz9260aj4w73r7jm747zm5fsgrdnrlpwy7rrwqjzlemm'
```

Comment:
change to actual multisig address deployed (axelar1ufs3tlq4umljk0qfe8k5ya0x6hpavn897u2cnf9k0en9jr7qarqqa9263g)